webserver/Models/Controllers/RealEstatesController.cs

```diff
@@ -0,0 +1,154 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using webserver.Data;
+using webserver.Models;
+using webserver.Models.DTOs;
+
+namespace webserver.Controllers;
+
+[ApiController]
+[Route("api/v1/realestates")]
```

Comment:
It seems all the routes are public.
So, you can do anything with no authentication.

I don't think we want that.

Reply:
Yeah i'll have to look further into this